packages/slice-machine/src/modules/slices/index.ts

```diff
@@ -144,7 +144,7 @@ export function* createSliceSaga({
   const addr = `/${payload.libName.replace(/\//g, "--")}/${
     payload.sliceName
   }/${variationId}`;
-  yield put(replace("/[lib]/[sliceName]/[variation]", addr));
+  yield put(push("/[lib]/[sliceName]/[variation]", addr));
```

Comment:
judging from the example, https://github.com/danielr18/connected-next-router/blob/dd523db706a8605cc199814ee1a1fa38e2f5b204/examples/basic/components/navigation.js

Push seems to be enough, is `take(Location_Change)` used as some kind of "dispatch" to tell Redux an event happened ?